src/main/java/com.vite/cli/ViteCLI.java

```diff
@@ -0,0 +1,69 @@
+package com.vite.cli;
+
+import com.vite.service.Client;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+
+public class ViteCLI {
+    private final BufferedReader in;
+    private final PrintWriter out;
+    private static final Logger LOGGER = LoggerFactory.getLogger(ViteCLI.class);
```

Comment:
Isn't what I have super expressive of what I am doing though? Also that looks very similar to reflection which can be evil especially when you don't need it